deployment/scripts/cluster.backup.sh

```diff
@@ -32,7 +32,7 @@ sleep 60
 # database backup
 kubectl --kubeconfig=${KUBECONFIG} -n default exec -it \
     $(kubectl --kubeconfig=${KUBECONFIG} -n default get pods | grep ocelot-neo4j | awk '{ print $1 }') \
-    -- neo4j-admin dump --to=/var/lib/neo4j/$BACKUP_DATE-neo4j-dump
+    -- neo4j-admin dump --database=graph.db --to=/var/lib/neo4j/$BACKUP_DATE-neo4j-dump
```

Comment:
We have to find out:

Does this work for Neo4j v3 as well?

If not we should have a seperate script for transition time. Or? @ulfgebhardt

Reply:
This is working for Neo4j v3.5 as well.
So lets merge it.